src/test/java/liquibase/ext/mongodb/statement/InsertOneStatementIT.java

```diff
@@ -20,34 +20,50 @@
  * #L%
  */
 
+import com.mongodb.MongoCommandException;
+import com.mongodb.MongoException;
+import com.mongodb.MongoWriteException;
 import com.mongodb.client.FindIterable;
 import com.mongodb.client.MongoDatabase;
 import liquibase.ext.AbstractMongoIntegrationTest;
+import liquibase.ext.mongodb.TestUtils;
 import org.bson.Document;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static liquibase.ext.mongodb.TestUtils.COLLECTION_NAME_1;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 
 class InsertOneStatementIT extends AbstractMongoIntegrationTest {
 
+    private String collectionName;
+
+    @BeforeEach
+    public void createCollectionName() {
+        collectionName = COLLECTION_NAME_1 + System.nanoTime();
+    }
+
     @Test
     void executeForObject() {
         final MongoDatabase database = connection.getDatabase();
         final Document document = new Document("key1", "value1");
-        new InsertOneStatement(COLLECTION_NAME_1, document, new Document()).execute(connection);
+        new InsertOneStatement(collectionName, document, new Document()).execute(connection);
 
-        assertThat(database.getCollection(COLLECTION_NAME_1).find())
-                .containsExactly(document);
```

Comment:
There is a subtlety to be aware of here:
 `org.bson.codecs.DocumentCodec` implements `org.bson.codecs.CollectibleCodec` which means that the driver will insert the `_id` on the driver side (method `generateIdIfAbsentFromDocument`) when the document is inserted. However the `runCommand` does not do this and the `_id` is generated by the server instead.
This shouldn't be a concern as both approaches generate a unique `ObjectId`. The test needed to be updated to reflect this.